src/components/LogInForm/LoginForm.js

```diff
@@ -18,12 +20,14 @@ const LoginForm = () => {
     resolver: yupResolver(schema),
   });
   const dispatch = useDispatch();
+  const { id, type, message } = failedLogin;
   const handleLogin = async (user) => {
     try {
       const userToLogin = await loginService.login(user);
       dispatch(logUserIn(userToLogin));
     } catch (error) {
       console.log(error);
+      toast[type](message, { id });
```

Comment:
I liked that you created a notifications file to store the messages. No magic string here 🙌  But it is a bit too much.

1- I think we don't need the id part. Why did you wanted to add the id to toast?
2- No need to store the type as well. Wee can just do toast.error(message). What do you think?

Reply:
1. In the docs, to prevent the same kind of toasts from piling up on each other, they are given an id. That is why I did it, otherwise as the user clicks on the login button with invalid credentials, an infinite amount of toasts of the same kind appear on the screen, which I thought would not be the preferred way. Should I keep it?
2. I thought that manually typing the toast's type may lead to unwanted outcomes if we have a very long list of notifications (writer may not be able to deduct the desired type of a notification if the variable name does not suggest it or simply write the wrong type accidentally). This is not the case for the failed login notification of course. However, I thought I should ensure that whenever "Geçersiz isim ya da şifre" is shown to the user, it should be an error message, which should not be changed in the component but changed in the constants.